tools/image_processing/fractal/cellvoyager_to_ome_zarr_init.xml

```diff
@@ -0,0 +1,152 @@
+<tool id="fractal_cellvoyager_to_ome_zarr" name="CellVoyager image to OME-Zarr" version="1.4.3" profile="24.2">
+    <description>Create OME-Zarr images from Cellvoyager images</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive">
+    <![CDATA[
+        mkdir -p tmp/input_images tmp/zarr_output tmp/outjson && 
+
+        #for $i, $filename in enumerate($input_images):
+            ln -s '$filename' 'tmp/input_images/${filename.element_identifier}' &&
+        #end for
+
+        version=`python3 --version | awk '{print tolower($1) substr($2,1,4)}'` &&
+        path=`which python3` &&
+        base="\${path%/*/*}" &&
+        new_path="\$base/lib/\$version/site-packages/fractal_tasks_core/tasks" &&
+        init_execute="cellvoyager_to_ome_zarr_init.py" &&
+        cli_init="\$new_path/\$init_execute" &&
```

Comment:
I think this will all go away if the software is in conda/containers and we have an entrypoint.

Reply:
Agreed! We'll look into this here: https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/945

Currently, the conda distribution anyway isn't what we'd want to use, as it only comes with the library part of the package, not all the dependencies for the individual tools (those are a pip extra we haven't exposed in conda so far).
We'll move away from using this package as a library in a future version of it and will then simplify multiple things. I've tagged exposing entry points as a task for that